.github/workflows/build_and_deploy.yml

```diff
@@ -18,22 +18,22 @@ jobs:
       runs-on: ubuntu-latest
       steps:
         -
-            name: Checkout
-            uses: actions/checkout@v3
+          name: Checkout
+          uses: actions/checkout@v3
         -
-            name: Login to Docker Hub
-            uses: docker/login-action@v2
-            with:
-                username: ${{ secrets.DOCKERHUB_USERNAME}}
-                password: ${{ secrets.DOCKERHUB_TOKEN}}
+          name: Login to Docker Hub
+          uses: docker/login-action@v2
+          with:
+            username: ${{ secrets.DOCKERHUB_USERNAME }}
+            password: ${{ secrets.DOCKERHUB_TOKEN }}
         -
-            name: Set up Docker Buildx
-            uses: docker/setup-buildx-action@v2
+          name: Set up Docker Buildx
+          uses: docker/setup-buildx-action@v2
         -
-            name: Build and Push
-            uses: docker/build-push-action@v4
-            with:
-                context: .
-                file: ./Dockerfile
-                push: true
-                tags: latest
+          name: Build and Push
+          uses: docker/build-push-action@v4
+          with:
+            context: .
+            file: ./Dockerfile
+            push: true
+            tags: justincely/pipsplace:latest
```

Comment:
ding ding.  This was it i'm pretty sure.  You were effectively doing `docker build -t latest` instead of `docker build -t justincely/pipsplace:latest`.  Then when it tried to push it tried to push `latest` which is a docker-level package like `ubuntu`.  Denied because it doesn't exist